cmd/zc_enumerator.go

```diff
@@ -200,6 +204,12 @@ func initResourceTraverser(source string, location common.Location, ctx *context
 			return nil, err
 		}
 
+		if sourceURL.Scheme == "http" {
```

Comment:
Safer perhaps to use EqualFold instead of ==?  (Not sure whether we can be sure that what we have in the URL object is always lower case.  Maybe they always are in Go. I don't know).

Reply:
Done